<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>230</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SettingsDialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>200</y>
     <width>158</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="okayButton">
      <property name="text">
       <string>Okay</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>-4</x>
     <y>-1</y>
     <width>411</width>
     <height>201</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="inputTab">
    <attribute name="title">
     <string>Input Tab</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>305</width>
       <height>88</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="mouseAlterate">
        <property name="text">
         <string>Use alterate mouse code (If mouse donset go to the right)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="anologSwitch">
        <property name="text">
         <string>Use Left anolog as mouse</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="virtualKeyboard">
        <property name="text">
         <string>Use virtual keyboard(W.I.P.)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="dPadMouse">
        <property name="text">
         <string>Use dPad as mouse (ComingSoon)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="generalTab">
    <attribute name="title">
     <string>GeneralTab</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>139</width>
       <height>65</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="autoStart">
        <property name="text">
         <string>Auto start</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="startMini">
        <property name="text">
         <string>Start minimized(W.I.P)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="useBlack">
        <property name="text">
         <string>Use black theme(W.I.P)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>okayButton</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>268</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>114</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
